virttest/utils_windows/drive.py

```diff
@@ -37,3 +39,93 @@ def get_floppy_drives_letter(session):
     """
     cond = "MediaType!=0 AND MediaType!=11 AND MediaType!=12"
     return _logical_disks(session, cond=cond, props=["DeviceID"])
+
+
+def rescan_disk(session):
+    """
+    Rescan disk in windows guest.
+
+    :param session: Session object.
+    """
+    rescan_cmd = "echo rescan > cmd"
+    rescan_cmd += " && echo exit >> cmd"
+    rescan_cmd += " && diskpart /s cmd"
+    session.cmd(rescan_cmd)
+
+
+def get_drive_volume_id(session, did, disk_letter):
+    """
+    Get a drive volume id in windows guest.
+
+    :param session: Session object.
+    :param did: Disk index.
+    :param disk_letter: Destination volume location.
+
+    :return: volume id, cmd header and cmd footer
+    """
+    list_disk_cmd = "echo list disk > cmd && "
+    list_disk_cmd += "echo exit >> disk && diskpart /s cmd"
+    disks = session.cmd_output(list_disk_cmd, timeout=120)
+
+    for disk in disks.splitlines():
```

Comment:
Well, if user passed `did`, it could directly select the certain disk by the given one, otherwise, it should look them up one by one.

Reply:
cut param did in func, just keep param session and disk_letter, thanks for your advise~